convert2rhel/pkghandler.py

```diff
@@ -365,6 +366,18 @@ def preserve_only_rhel_kernel():
     return
 
 
+def install_gpg_keys():
+    loggerinst = logging.getLogger(__name__)
+    gpg_path = os.path.realpath(os.path.join(os.path.dirname(__file__),
+                                "data/version-independent/gpg-keys/*"))
+    for gpg_key in glob.glob(gpg_path):
```

Comment:
```suggestion
    gpg_keys = [os.path.join(gpg_path, x) for x in os.listdir(gpg_path)]
    for gpg_keys:
```
Similarly to:
https://github.com/oamg/convert2rhel/blob/master/convert2rhel/subscription.py#L137